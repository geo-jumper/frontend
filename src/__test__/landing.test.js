import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow, render, mount } from 'enzyme';


Enzyme.configure({adapter: new Adapter()});

import Landing from '../components/landing';

describe('Landing', () => {
  test('render a heading', () => {
    const wrapper = shallow(
      <h1> Welcome to Geo-Jumper </h1>
    );
    expect(wrapper).toMatchSnapshot();
  });

  test('render a div id', () => {
    const wrapper = shallow(
      <div id="landing"></div>
    );
    expect(wrapper.prop('id')).toEqual('landing');
  });
});


