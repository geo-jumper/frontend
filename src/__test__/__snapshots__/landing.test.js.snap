// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing render a heading 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <h1>
     Welcome to Geo-Jumper 
  </h1>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": " Welcome to Geo-Jumper ",
    },
    "ref": null,
    "rendered": " Welcome to Geo-Jumper ",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " Welcome to Geo-Jumper ",
      },
      "ref": null,
      "rendered": " Welcome to Geo-Jumper ",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
